course-info.yaml

```diff
@@ -16,4 +16,4 @@ content:
   - introductionSection
   - psiSection
 environment_settings:
-  jvm_language_level: JDK_17
+  jvm_language_level: JDK_19
```

Comment:
Looks like we can leave `JDK_17`:
https://github.com/jetbrains-academy/intellij-platform-plugin-course/pull/27#issue-2716078512

Reply:
I think it's a plugin bug
It seems the plugin doesn't take into account existing value and always rewrites this value according to your JDK

I intentionally changed it to 17 since plugin development requires 17 and there is no need to narrow down that range of "valid" JDK for this course